doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -155,10 +155,105 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 ###<a name="section3.2"></a>Using a specific configuration
 As seen above, the default configuration distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
 
-* It only works for building historical context data in MySQL.
-* The endpoint for MySQL is `mysql`.
-* The logging level is `INFO`.
-* The logging appender is `console`.
+* MySQL:
+  * It only works for building historical context data in MySQL.
+  * The user for MySQL is `mysql`.
+  * The pass for MySQL is `mysql`.
+  * The logging level is `INFO`.
```

Comment:
If I remember well, we discussed offline about adding a 5th main bullet regarding log4j configuration, which is cross configuration to all the sinks.